includes/classes/ExternalConnections/WordPressExternalConnection.php

```diff
@@ -543,7 +543,7 @@ public function push( $post_id, $args = array() ) {
 		 * Now let's push
 		 */
 		$post_body = [
-			'title'                          => get_the_title( $post_id ),
+			'title'                          => html_entity_decode( get_the_title( $post_id ) ),
```

Comment:
@dinhtungdu Wondering if we should pass in the character set we want to use for decoding? I think by default it is UTF-8, which is probably good for most use cases, but some sites may use a different encoding.

Looking at core, most places they use `html_entitiy_decode`, they have it set up like this: `html_entity_decode( $content, ENT_QUOTES, get_bloginfo( 'charset' ) )`

Reply:
Nice catch! Thank you!